Modules/System/AL Rest Client/src/Authentication/HttpAuthenticationBasic.Codeunit.al

```diff
@@ -0,0 +1,47 @@
+/// <summary>Implementation of the "Http Authentication" interface for a request that requires basic authentication</summary>
+codeunit 50304 "Http Authentication Basic" implements "Http Authentication"
+{
+    var
+        GlobalUsername, GlobalPassword : Text;
```

Comment:
I think these globals should also be tagged as NonDebuggable

Reply:
I totally agree.